### DISCUSSION TITLE
Changing SQL Server IP

### DISCUSSION THREAD

**Discussion by nicosuter** [modoboa/modoboa#2370] [Help] [answered]
Hey everyone,

Is there a single command that changes the MySQL server IP in all modoboa-related files?

The database server has recently changed its IPs and from what I can currently tell, I'd need to change the IP in every config file manually.

Is this correct? If yes, could someone send all the file paths incase I forget any?

**Comment by tonioo** [comment, marked as answer]
@RyderForReal There is no command to update the IP address everywhere. You can start by updating the settings.py file and the rerun the generate_postfix_maps command to update postfix configuration (see https://modoboa.readthedocs.io/en/latest/manual_installation/postfix.html#map-files). Then, update dovecot configuration (see https://modoboa.readthedocs.io/en/latest/manual_installation/dovecot.html#mysql-users), opendkim configuration if you use it (https://modoboa.readthedocs.io/en/latest/manual_installation/opendkim.html#database) and amavis configuration if you use it (https://modoboa-amavis.readthedocs.io/en/latest/setup.html)

**Reply by nicosuter** [reply]
Alright, thank you!
